### DISCUSSION TITLE
When to make version 2.0

### DISCUSSION THREAD

**Discussion by scotthibbs** [w7sst/MorseRunner#162] [General]
[w7sst](https://github.com/w7sst) [6 hours ago](https://github.com/w7sst/MorseRunner/pull/161#discussion_r1038971617)
Hopefully we will roll to Version 2.0 soon, then 2.0+ can represent to CE versions into the future. Perhaps we should get to Version 2.0 sooner than later. I was hoping to wait until N1MM integration was achieved before considering a big version number change.

I wonder if we should be considering this release as 2.0? Perhaps the 1.8x series was based on the prototype and now that we are ready to announce this things, we can call it 2.0? I don't know if we are ready or not.

Owner
@[w7sst](https://github.com/w7sst) w7sst [6 hours ago](https://github.com/w7sst/MorseRunner/pull/161#discussion_r1038971788)
Perhaps waiting for a future release before rolling to 2.0, not now.

**Comment by scotthibbs** [comment]
With the changes to the GUI that we proposed, it will look like a "new" Morse Runner. 
Probably will be able to do these changes during the rework with N1MM integration so both these changes it will feel appropriate to do 2.0.

**Comment by w7sst** [comment]
I am also thinking we will use version numbers
- 1.9x to represent the N1MM-compatible versions. This will hopefully be released in the spring.
- 2.x.y will be for the new GUI releases sometime next year, probably fall.
